### DISCUSSION TITLE
I can't see my coin..

### DISCUSSION THREAD

**Discussion by eugenetsou** [Chia-Network/chia-blockchain#4837] [Support]
Hi Chia Support Team

I have 4 private key(ex. A,B,C,D) and I have Key A,B,C's plots farming on 3 different computers, and setup the [Farm]\[Manage You Farming Rewards Target Address] to the Key D's [Receive Address] in all "Chia Blockchain" program.

Today, I check my key D's [Receive Address] in chia explorer, find out I won a block, but I can see nothing in my wallets, Is it because I set the wrong and lost the coins?

Please advise how to deal with, Thank you!

**Comment by eugenetsou** [comment]
just find out my wallet is not synced..........

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
